crow/timeline.md

```diff
@@ -255,8 +255,8 @@ timeline.score{
   }
 ```
 
-above it was stated that the "reset" message has to happen at the *end* of the timeline. this is not strictly true! any event can return the "reset" message and jump to the start of the timeline. exploiting this could lead to some very interesting repeating sequences that play different amounts of the whole score...
+It was stated above that the "reset" message has to happen at the *end* of the timeline. This is not strictly true! Any event can return the "reset" message and jump to the start of the timeline. Exploiting this could lead to some very interesting repeating sequences that play different amounts of the whole score.
 
-## wrapping up
+## Wrapping up
 
-there's a lot of features here, but timeline is all about making rhythmic ideas concise. you'll likely need the extra features only occasionally, so try and build up ideas slowly. adding a sequins or two into your timeline is a fantastic way to add variation into your patterns!
+There are a lot of features here to absorb, but timeline is all about making rhythmic ideas more concise. You'll likely need many of timelines features only occasionally, so try and build up your ideas slowly. And remember that sprinkling a sequins or two into your timeline is a fantastic way to add variation into your patterns!
```

Comment:
many of timelines features -> many of `timeline`'s features
and i guess generally @trentgill -- do we want to wrap _any_ mention of sequins or timeline or score in ticks, so it stands out as code?

Reply:
i was certainly doing this inconsistently. honestly we could go either way and i have no horse in that race.